### ISSUE TITLE
Exec reporting incorrect exit code

### ISSUE THREAD

**Issue by dvkt** [Lartu/ldpl#91]
Tested this on Mac and Linux, seems like the exit code is getting misreported: 

```
$ cat report.ldpl
DATA:
code is number
PROCEDURE:
execute "sh bad.sh" and store exit code in code
display "exit code: " code crlf

$ cat bad.sh
#!/bin/sh
exit 1

$ ldpl report.ldpl
LDPL: Compiling...
* File(s) compiled successfully.
* Saved as report-bin

$ ./report-bin
exit code: 256

$ sh bad.sh
$ echo "exit code: $status"
exit code: 1
```

**Comment by Lartu** [comment]
Hmmm, that's weird. At the moment I'm very busy studying for a final, but once it's passed I'll check this out. Thank you for the report!

**Comment by Lartu** [comment]
Fixed in 6600458d35bb04061e218dacb1aef458d610d820!
